### ISSUE TITLE
No such container: dokku.tor

### ISSUE THREAD

**Issue by ohld** [michaelshobbs/dokku-tor#2]
If you run dokku-tor on modern Dokku server according to current instructions in Readme.md, this error produces:

```
dokku ps:restart my_dokku_app
-----> Deploying web (count=1)
Error response from daemon: could not get container for dokku.tor: No such container: dokku.tor
parallel: This job failed:
/var/lib/dokku/plugins/available/scheduler-docker-local/bin/scheduler-deploy-process-container gate herokuish dokku/gate:latest latest web 1 1
 !     exit status 1
```

Plugin was installed successfully though.

**Comment by ohld** [comment]
will be fixed with #3

**Comment by ohld** [comment]
@michaelshobbs by the way. Once I had dokku-tor successfully installed I started to see this issue while deploying other dokku apps. 

```
/var/lib/dokku/plugins/enabled/tor/commands: line 85: is_container_running: command not found
```

**Comment by rct-k** [comment]
@ohld take a look at the fix i added in #4
